packages/define-emit/package.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
I think we can combine these two packages into one package. `@vue-macros/single-define`

Reply:
ok, I will make the changes and send till next Saturday